portal-api/src/v1/controllers/companies.controller.js

```diff
@@ -0,0 +1,13 @@
+const externalApi = require('../../external-api/api');
+
+const getCompanyByRegistrationNumber = async (req, res) => {
+  const { registrationNumber } = req.params;
+
```

Comment:
* `registrationNumber` void check please.
* Wrap in `try-catch` please.

Reply:
Done.